tests/integration/conftest.py

```diff
@@ -46,11 +62,15 @@ async def charm(ops_test: OpsTest) -> str:
     When multiple charm files (i.e., for different bases) are produced by a `charmcraft pack`,
     our CI will currently set the variable to the highest-base one.
     """
+    original_text = '"level": "WARN"'
+    modified_text = '"level": "INFO"'
+    change_text_in_file(CONFIG_BUILDER_PATH, original_text, modified_text)
```

Comment:
This won't work in CI, as the charm is already packed and passed via `CHARM_PATH`; we should do this modification on the live config rather in the charm source

Reply:
Doing this in the live config is not working since every time a new event is received the config is overwritten.